spec/spec_helper.rb

```diff
@@ -131,6 +131,8 @@
   end
 end
 
+RSpec::Matchers.define_negated_matcher :not_change, :change
```

Comment:
"not to change" chained with "and change" is ambiguous (is it supposed to not change, the second time, or to change?) and rspec throws an error telling me to define a negative matcher to make it more explicit.